File: C files/logicalOperators.c
#include <stdio.h>
#include <stdbool.h>

int main() {

/*    // logical operators = && (AND) checks if two conditions are true

    float temp = 25;
    bool sunny = false;

    if(temp >= 0 && temp <=30 && sunny == true){
        printf("\nThe weather is good");
    }
    else{
        printf("\nThe weather is bad");
    }
*/
    

/*    // logical operators = || (OR) checks if at least one condition is true
    float temp = 25;
    bool sunny = false;

    if(temp <= 0 || temp >= 30 || sunny == false){
        printf("\nThe weather is bad");
    }
    else{
        printf("\nThe weather is good");
    }
*/

    // logical operators = ! (NOT) reverses the state of a condition

   bool sunny = false;
   
   if(!sunny == true){
    printf("\nIt's cloudy outside");
   }
   else{
    printf("\nIt's sunny outside");
   }


return 0;
}

File: Python files/logical_operators.py
# logical operators = evaluate multiple conditions (or, and, not)
#                     or = at least one condition must be TRUE
#                     and = both conditions must be TRUE
#                     not = inverts the condition (not FALSE, not TRUE)

# temp = 25
# is_raining = False

# if temp > 35 or temp < 0 or is_raining:
#     print("The outdoor event is cancelled!")
# else:
#     print("The outdoor event is still scheduled!")

temp = 15
is_sunny = False

if temp >= 28 and is_sunny:
    print("It is hot outside")
    print("It is sunny")
elif temp <= 0 and is_sunny:
    print("It is cold outside")
    print("It is sunny")
# elif temp < 28 and temp > 0 and is_sunny:
#     print("It is warm outside")
#     print("It is sunny")
elif 28 > temp > 0 and is_sunny:
    print("It is warm outside")
    print("It is sunny")
elif temp >= 28 and not is_sunny:
    print("It is hot outside")
    print("It is cloudy")
elif temp <= 0 and not is_sunny:
    print("It is cold outisde")
    print("It is cloudy")
elif 28 > temp > 0 and not is_sunny:
    print("It is warm outside")
    print("It is cloudy")
    